.github/workflows/test-runner-on-perf.yml

```diff
@@ -1,230 +0,0 @@
-          cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-helm-definitions/perf
-          echo "Apply helm updates, including updated data-repo version"
-          helmfile apply
-          cd ${GITHUB_WORKSPACE}/${workingDir}
-      - name: "[Clear Perf Database] Import Perf Database Secret"
-        uses: hashicorp/vault-action@v2.5.0
-        with:
-          url: ${{ secrets.VAULT_ADDR }}
-          method: approle
-          roleId: ${{ secrets.PERF_ROLE_ID }}
-          secretId: ${{ secrets.PERF_SECRET_ID }}
-          #Yes, they both point to the stairway database password
-          secrets: |
-            secret/dsde/datarepo/perf/datarepo-sql-db stairwaypassword | ORG_GRADLE_PROJECT_dbDatarepoPassword ;
-            secret/dsde/datarepo/perf/datarepo-sql-db stairwaypassword | ORG_GRADLE_PROJECT_dbStairwayPassword ;
-      - name: "[Clear Perf Database] Perform DropAll"
-        env:
-          ORG_GRADLE_PROJECT_dbDatarepoUri: jdbc:postgresql://localhost:5434/datarepo
-          ORG_GRADLE_PROJECT_dbStairwayUri: jdbc:postgresql://localhost:5434/stairway
-        run: |
-          # port foward to perf database
-          kubectl port-forward --namespace perf svc/perf-jade-gcloud-sqlproxy 5434:5432 &
-          PID=$!
-          sleep 10
-          echo "After giving the proxy 10 seconds to spin up, run drop all on perf database"
-          cd ${GITHUB_WORKSPACE}/${workingDir}
-          ./gradlew dropAll
-          kill ${PID}
-      - name: "[Clear Perf Database] Cycle pods to initialize databases"
-        run: |
-          echo "Cycle API pods to trigger migrations to initialize databases"
-          kubectl delete pods -n perf -l app.kubernetes.io/name=datarepo-api
-          echo "Sleep 45 seconds to give the pods a chance to start cycling before checking if up and on correct version"
-          sleep 45
-      - name: "Wait for Perf Cluster to come back up with correct version"
-        uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.69.0
-        env:
-          NAMESPACEINUSE: perf
-          IT_JADE_API_URL: "https://jade-perf.datarepo-perf.broadinstitute.org"
-          DESIRED_GITHASH: ${{ steps.read_property.outputs.LATEST_GITHASH }}
-          DEPLOYMENT_TYPE: 'api'
-      - name: "Build and run Test Runner"
-        run: |
-          cd ${GITHUB_WORKSPACE}/${workingDir}
-          echo "Building Data Repo client library"
-          export TEST_RUNNER_SERVER_SPECIFICATION_FILE="perf.json"
-          ENABLE_SUBPROJECT_TASKS=1 ./gradlew :datarepo-client:clean :datarepo-client:assemble
-          cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-clienttests
-          export ORG_GRADLE_PROJECT_datarepoclientjar=$(find .. -type f -name "datarepo-client*.jar")
-          echo "ORG_GRADLE_PROJECT_datarepoclientjar = ${ORG_GRADLE_PROJECT_datarepoclientjar}"
-
-          echo "Running test suite"
-          ./gradlew runTest --args="suites/NightlyPerfWorkflow.json tmp/TestRunnerResults"
```

Comment:
I think that we also should delete
`datarepo-clienttests/src/main/resources/suites/NightlyPerfWorkflow.json` -- right now I think only `datarepo-clienttests/src/main/resources/measurementlists/NightlyPerfWorkflow.json` is deleted!

Reply:
Oh I added that back in case we wanted to run that test suite manually, but I supposed we could run tests individually as well

Reply:
Ah!  No that's a good point, I think that makes sense.

Reply:
Ok great! I'll make a separate ticket for spinning down the perf environment